metrics/web/lib/base/presentation/widgets/material_container.dart

```diff
@@ -0,0 +1,68 @@
+// Use of this source code is governed by the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+/// A widget that represents the material container.
+class MaterialContainer extends StatelessWidget {
+  /// A [Widget] below this in the widget tree.
+  final Widget child;
+
+  /// An amount of space by which to inset the [child].
+  final EdgeInsets padding;
+
+  /// A [MaterialType] of this container.
+  final MaterialType type;
+
+  /// A z-coordinate at which to place this material widget related to it's
+  /// parents.
+  final double elevation;
+
+  /// A background [Color] of this widget.
```

Comment:
```suggestion
  /// A background [Color] of this container.
```
Would it be more consistent to replace all "this widget" mentions with "this container"?